.github/workflows/linux_and_macos.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             clang_repo_suffix: null
             make: bmake
             runs-on: ubuntu-22.04
-          - cc: gcc-11
+          - cc: gcc-12
             cxx: g++-11
```

Comment:
Please also adjust this `g++-11` line. Or make a dedicated commit upfront deleting `cxx` altogether. What do you think?

Reply:
It looks to me like `cxx` is never used. I will remove it in a commit before this one (maybe after dinner).

Reply:
@edgar-bonet thanks! :+1: